src/components/DownloadManifestModal.js

```diff
@@ -88,13 +95,16 @@ const GenerateManifestSet = compose(injectState, withTheme)(
               copyValueToClipboard({ value: setId, copyRef });
             } else {
               const type = 'file';
-              const { data, errors } = await saveSet({
-                type,
-                sqon: sqon || {},
-                userId: loggedInUser.egoId,
-                path: 'kf_id',
-                api: graphql(api),
-              });
+              const [{ data, errors }] = await Promise.all([
+                saveSet({
+                  type,
+                  sqon: sqon || {},
+                  userId: loggedInUser.egoId,
+                  path: 'kf_id',
+                  api: graphql(api),
+                }),
+                wait(1000),
```

Comment:
it seems the request tends to be so fast that it looks like a flash so we're putting in an arbitrary delay